packages/manager/modules/web-paas/src/components/additional-option/template.html

```diff
@@ -0,0 +1,45 @@
+<div class="oui-tile d-flex m-2">
+    <h4 class="oui-tile__title ng-binding w-50">
```

Comment:
Why limit the width ?

Reply:
Its breaking the next element `price` if the title is too long. Limiting width pushes the title to next line and makes sure the price text does not break